utils/python/make_nexus_output_pretty.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
+__author__ = 'Barry Baker'
+__email__ = 'barry.baker@noaa.gov'
+__license__ = 'GPL'
+
+import netCDF4 as nc
+import numpy as np
+from netCDF4 import date2num,num2date
+import datetime as dt
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter 
+
+def get_hemco_dates(time_file='HEMCO_sa_Time.rc'):
+    fi = open(time_file)
+    lines = fi.readlines()
+    fi.close()
+    for L in lines:
+        if L.startswith('START'):
+            start_base = dt.datetime.strptime(L.split()[1],'%Y-%m-%d')
+            start = dt.datetime.strptime(L, 'START:   %Y-%m-%d %H:00:00\n')
```

Comment:
Seems kind of fragile with the spacing. Could so something like this
```suggestion
            start = dt.datetime.strptime(' '.join(L.split()[1:3]), '%Y-%m-%d %H:00:00')
```